docs/general/performance.md

```diff
@@ -59,6 +59,10 @@ new Chart(ctx, {
 });
 ```
 
+## Data structure and format
+
+Provide prepared data in the internal format accepted by the dataset and scales. See [Data structures](data-structures.md) for more information.
```

Comment:
Maybe change to "Provide prepared data in the internal format accepted by the dataset and scales and set `parsing: false`."

Reply:
done